proto/provenance/metadata/v1/scope.proto

```diff
@@ -84,12 +84,12 @@ message Scope {
   ];
   // These parties represent top level owners of the records within.  These parties must sign any requests that modify
   // the data within the scope.  These addresses are in union with parties listed on the record groups.
-  repeated string owner_address = 3;
+  repeated Party parties = 3 [(gogoproto.nullable) = false];
```

Comment:
Thoughts on leaving this field named `owners` here but using the updated `Party` structure?

Reply:
I got about 1/2 way through and had the same thought. I'll change it to `owners`.